src/realm/sync/client.cpp

```diff
@@ -812,9 +813,7 @@ void SessionImpl::process_pending_flx_bootstrap()
             },
             get_transact_reporter());
         progress = *pending_batch.progress;
-
-        REALM_ASSERT(call_debug_hook(SyncClientHookEvent::DownloadMessageIntegrated, progress, query_version,
```

Comment:
Why is this removed? I am using it in RCORE-1225 to update an object brought into view during bootstrap.

Reply:
I think by accident. I've put it back.